### ISSUE TITLE
Descriptions of the Custom CSRs

### ISSUE THREAD

**Issue by palmer-dabbelt** [rjiejie/XuanTie-doc#1]
Thanks for posting documentation for this, and sorry I hadn't seen it earlier (Vineet just pointed me to it, but I guess it'd been posted on some of the lists).  I was poking around the docs and notices some custom CSR bits (`mxstatus.theadisaee`, for example), but don't see any definition of what those do.

**Comment by rjiejie** [comment]
Yes, all extended CSRs of T-HEAD do not exist in the doc, and we will update in the next version,
thanks for your review :)

**Comment by orangecms** [comment]
Hi, I've looked a bit into those and dropped some notes here while working on oreboot and RustSBI for the Nezha board:
https://github.com/oreboot/oreboot/pull/519

Google Translate worked quite okay for that purpose. :-)

**Comment by btashton** [comment]
@rjiejie any chance there is a updated version with these around? I am slowly documenting on my own based on the Chinese manual, but it is slow as I am not a native speaker.

**Comment by rjiejie** [comment]
> @rjiejie any chance there is a updated version with these around? I am slowly documenting on my own based on the Chinese manual, but it is slow as I am not a native speaker.

You could get the latest document from https://github.com/T-head-Semi/thead-extension-spec,
all of the open-source T-Head ISAs will been updated there :)
